me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -1,51 +1,54 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Well, this one did. :wink:  And also this happened before with other files. If you try to normalize the line endings, and there are changes, than some files used CRLF (see the end of the previously referenced GH article).